app/jobs/runtime/pollable_job_cleanup.rb

```diff
@@ -5,10 +5,10 @@ class PollableJobCleanup < VCAP::CloudController::Jobs::CCJob
         CUTOFF_AGE_IN_DAYS = 90
 
         def perform
-          old_orphaned_blobs = PollableJobModel.where(Sequel.lit("created_at < CURRENT_TIMESTAMP - INTERVAL '?' DAY", CUTOFF_AGE_IN_DAYS))
-          logger = Steno.logger('cc.background.expired-orphaned-blob-cleanup')
-          logger.info("Cleaning up #{old_orphaned_blobs.count} OrphanedBlob rows")
-          old_orphaned_blobs.delete
+          old_pollable_jobs = PollableJobModel.where(Sequel.lit("created_at < CURRENT_TIMESTAMP - INTERVAL '?' DAY", CUTOFF_AGE_IN_DAYS))
+          logger = Steno.logger('cc.background.expired-pollable-jobs-cleanup')
```

Comment:
I have one tiny wish: when fixing this we should also remove the `expired` und use singular `job`, so that the logger name matches the class name, i.e. `cc.background.pollable-job-cleanup`.

Reply:
Done